manifest.json

```diff
@@ -1,17 +1,17 @@
 {
   "name": "KillProof Module",
-  "version": "1.4.4",
+  "version": "1.4.5",
   "namespace": "Nekres.KillProof",
   "package": "KillProof Module.dll",
   "manifest_version": 1,
 
   "description": "Check someone's kills for raiding. Uses ArcDPS Bridge service if available.",
   "dependencies": {
-    "bh.blishhud": ">=0.4.4"
+    "bh.blishhud": ">=0.4.6"
```

Comment:
I think this should actually be 0.5.0 since you've updated Gw2Sharp.

Reply:
oh sry I thought the gw2sharp update was 0.4.6

Reply:
No worries.  😄